frontend/src/lib/apiClient.js

```diff
@@ -14,6 +14,30 @@ const getRecipes = (search) => {
     .then(results => results.data);
 };
 
+const createUser = data => apiClient.post('api/v1/user', {
+  user: {
+    first_name: data.firstName,
+    last_name: data.lastName,
+    email: data.email,
+    password: data.password,
+  },
+});
+
+const loginUser = (data) => {
+  const credentials = { email: data.email, password: data.password };
+  return apiClient.post('api/v1/auth/login', {
+    user: credentials,
+  });
+};
+
+const getUser = email => apiClient.get('api/v1/user', {
```

Comment:
This is one of those situations where the user ID is going to be helpful. A GET request is probably going to ignore whatever's in the body. But if you `GET /api/v1/user/1`, that'll get the user info for whoever's primary key is 1. So instead of passing in the email, we may want to pass in the user_id.

In this case, you'll only be able to get that info if you're sending a token authorized to view that user, so you'll want to send along the JWT with this request, as well. You'd put that in the `Authorization` header.